src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py

```diff
@@ -0,0 +1,138 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+import json
+from json import JSONDecodeError
+from pathlib import Path
+from typing import Optional, Dict, Union, Callable
+
+import numpy as np
+import pandas as pd
+
+from ._utils import load_jsonl
+from ._flow_run_wrapper import FlowRunWrapper
+from promptflow import PFClient
+
+
+def _calculate_mean(df) -> Dict[str, float]:
+    mean_value = df.mean(numeric_only=True)
+    return mean_value.to_dict()
+
+
+def _load_data(data: str) -> Union[str, Path]:
+    try:
+        if isinstance(data, (str, Path)):
+            if Path(data).exists():
+                test_data = load_jsonl(data)
+                _data_is_file = True
```

Comment:
`_data_is_file` is not being referenced anywhere. I think it can be just removed.

Reply:
Thanks good catch will remove it.